src/quo2/components/tags/status_tags.cljs

```diff
@@ -61,9 +64,10 @@
                    :color        text-color}} label]]])))
 
 (defn- positive
-  [size theme label _ no-icon?]
+  [size theme label _ no-icon? container-style]
```

Comment:
preferably these args are a map instead

Reply:
This would affect the rest of the component and would need further changes for all the other cases. This is not part of this PR, I just needed to add a container-style